packages/core/src/types.ts

```diff
@@ -73,12 +74,21 @@ export type BundlerReport = {
 
 export type ToInjectItem = { type: 'file' | 'code'; value: string; fallback?: ToInjectItem };
 
+export type Logger = {
+    (text: any, type?: LogLevel): void;
```

Comment:
why do you need this type?

Reply:
It was to cover legacy usage.
But agreed (with your other comment as well), while I'm in there,
I'll remove this usage entirely.